patches/ui-native_theme-dark_mode_observer.h.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/ui/native_theme/dark_mode_observer.h b/ui/native_theme/dark_mode_observer.h
+index 0f2c79e36e3dfc96621a9d7b01f8a3c9f9e05640..fe0e1c83152e279f9d39b2b5480acaef474a6c6a 100644
+--- a/ui/native_theme/dark_mode_observer.h
++++ b/ui/native_theme/dark_mode_observer.h
+@@ -23,7 +23,7 @@ class NATIVE_THEME_EXPORT DarkModeObserver : public NativeThemeObserver {
+   // Observe |theme| for changes in dark mode. |callback_| may be immediately
+   // run if |theme_->SystemDarkModeEnabled()| has changed since construction or
+   // last time |Stop()| was called.
+-  void Start();
++  virtual void Start();
```

Comment:
what is our DarkModeObserver actually doing? Is it still necessary? Please cc code owner because I think we can use `OnNativeThemeUpdated` instead and this patch will not be necessary

Reply:
ok, looks like @simonhong came back later and cleaned this up

Reply:
yup! fixed with https://github.com/brave/brave-core/pull/2829/commits/76bc85748f498907a8d412eb53b961039d1df032